extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -304,6 +304,23 @@ public function getWikiVariables() {
 	}
 
 	/**
+	 * @desc Returns redirected article or null
+	 *
+	 * @return mixed
+	 * @throws BadRequestApiException
+	 * @throws NotFoundApiException
+	 */
+	public function getRedirectedTitleFromRequest() {
+		$title = $this->getTitleFromRequest();
```

Comment:
You're calling `getTitleFromRequest` inside this function and outside - please refactor this.